prow/config/config.go

```diff
@@ -1943,6 +1943,14 @@ func parseProwConfig(c *Config) error {
 		return fmt.Errorf("tide has invalid max_goroutines (%d), it needs to be a positive number", c.Tide.MaxGoroutines)
 	}
 
+	if len(c.Tide.TargetURLs) > 0 && c.Tide.TargetURL != "" {
+		return fmt.Errorf("tide.target_url and tide.target_urls are mutually exclusive")
+	}
+
+	if c.Tide.TargetURLs == nil {
+		c.Tide.TargetURLs = map[string]string{}
+	}
```

Comment:
I think you also want to copy c.Tide.TargetURL into ci.Tide.TargetURLs if the former is non-empty

Reply:
Fixed and added unit tests.